docs/flashbots-auction/advanced/rpc-endpoint.mdx

```diff
@@ -768,6 +768,76 @@ where
 - `consideredByBuildersAt`: indicates time at which each builder selected the bundle to be included in the target block
 - `sealedByBuildersAt`: indicates time at which each builder sealed a block containing the bundle
 
+### flashbots_getSbundleStats
+
+The `flashbots_getSbundleStats` JSON-RPC method returns stats for a single sbundle. You must provide bundleHash, and the signing address must be the same as the one who submitted the bundle.
```

Comment:
1. Can we define what "sbundle" means? Or link to an existing description?
2. Nit: Format "bundleHash" as code (ie. using the slash quotes like: `bundleHash`)